dev-libs/mini-xml/mini-xml-2.12.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+MY_P="${P/mini-xml/mxml}"
+
+DESCRIPTION="A small XML parsing library that you can use to read XML data files or strings"
+HOMEPAGE="https://github.com/michaelrsweet/mxml
+	https://www.msweet.org/mxml/"
+SRC_URI="https://github.com/michaelrsweet/mxml/releases/download/v${PV}/${MY_P}.tar.gz"
```

Comment:
there is standard https://github.com/michaelrsweet/mxml/archive/v2.12.tar.gz

so I do not know why rhe whole ranaming game is needed. but `-> ${P}.tar.gz`

Reply:
Hm, good question. Thanks, changed now.